src/modules/scraper/adapter/messaging/FillEventsConsumer.ts

```diff
@@ -45,7 +45,14 @@ export class FillEventsConsumer {
       throw new Error("Deposit not found in db");
     }
 
-    if (deposit.fillTxs.includes({ hash: transactionHash, totalFilledAmount })) return;
+    const fillTxIndex = deposit.fillTxs.findIndex(
+      (fillTx) => fillTx.hash === transactionHash && fillTx.totalFilledAmount === totalFilledAmount,
+    );
+
+    if (fillTxIndex !== -1) {
+      this.logger.warn("Fill event already processed");
+      return;
+    }
```

Comment:
`includes` method is not suitable for checking the existing of an object inside an array. I changed it with `findIndex`

Reply:
Why wouldn't included work?